src/functionHandlers/notarisePdt/v2/handler.ts

```diff
@@ -64,6 +64,18 @@ export const notarisePdt = async (
   let signedEuHealthCerts: notarise.SignedEuHealthCert[] = [];
   if (config.isOfflineQrEnabled || whiteListNrics.includes(patientNricFin)) {
     try {
+      const testDataTypes = testData.map((test) => test.swabTypeCode);
+      if (
+        !testDataTypes.includes(config.swabTestTypes.ART) &&
+        !testDataTypes.includes(config.swabTestTypes.PCR)
+      ) {
```

Comment:
Should we check for single type vs multi type first? Since EU HealthCert only supports single type only. Because this if statement might still miss out `["PCR", "ART"]` (invalid).

Can do something similar to this: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/eb9d0d77e85adb9ddf00193bb35ad9e704e82a4b/src/functionHandlers/notarisePdt/validateInputs/validateDocument.ts#L131-L134

Reply:
My bad, misunderstood previously.

So EU HealthCert supports:
- `"PCR"`
- `"ART"`
- `["PCR", "SER"]`